Tests/SonarScanner.MSBuild.Shim.Test/ScannerEngineInputGeneratorTest.cs

```diff
@@ -41,11 +42,13 @@ public void Constructor_Null_Throws()
         var cnfg = new AnalysisConfig();
         var rntm = runtime;
         var rvsf = new RoslynV1SarifFixer(runtime.Logger);
-        FluentActions.Invoking(() => new ScannerEngineInputGenerator(null, rntm)).Should().ThrowExactly<ArgumentNullException>().WithParameterName("analysisConfig");
-        FluentActions.Invoking(() => new ScannerEngineInputGenerator(cnfg, null)).Should().ThrowExactly<ArgumentNullException>().WithParameterName("runtime");
-        FluentActions.Invoking(() => new ScannerEngineInputGenerator(cnfg, null, null, null)).Should().ThrowExactly<ArgumentNullException>().WithParameterName("runtime");
-        FluentActions.Invoking(() => new ScannerEngineInputGenerator(cnfg, rntm, null, null)).Should().ThrowExactly<ArgumentNullException>().WithParameterName("fixer");
-        FluentActions.Invoking(() => new ScannerEngineInputGenerator(cnfg, rntm, rvsf, null)).Should().ThrowExactly<ArgumentNullException>().WithParameterName("additionalFilesService");
+        var prvdr = new ListPropertiesProvider();
```

Comment:
The trick is that all of them has the same lenght as `null` for a proper table layout
```suggestion
        var cmds = new ListPropertiesProvider();
```